network-store-model/src/main/java/com/powsybl/network/store/model/DanglingLineAttributes.java

```diff
@@ -94,4 +95,64 @@ public class DanglingLineAttributes extends AbstractAttributes implements Inject
 
     @Schema(description = "Active power limits")
     private LimitsAttributes activePowerLimits;
+
+    @Override
+    @JsonIgnore
+    public LimitsAttributes getLimits(TemporaryLimitType type, int side) {
+        switch (type) {
+            case CURRENT_LIMIT:
+                if (side == 1) {
```

Comment:
Test `side` argument only once at the beginning of the function, then return the correct `LimitsAttributes` instance in the switch

Reply:
Done